examples/flutter_gallery/lib/demo/expansion_panels_demo.dart

```diff
@@ -0,0 +1,328 @@
+// Copyright 2016 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter/material.dart';
+
+enum _Location {
+  Barbados,
+  Bahamas,
+  Bermuda
+}
+
+class DualHeaderWithHint extends StatelessWidget {
+  DualHeaderWithHint({
+    this.name,
+    this.value,
+    this.hint,
+    this.showHint
+  });
+
+  final String name;
+  final String value;
+  final String hint;
+  final bool showHint;
+
+  Curve _flipCurves(bool isExpanded) {
+    return isExpanded ? new Interval(0.0, 0.6, curve: Curves.ease) :
+        new Interval(0.4, 1.0, curve: Curves.ease);
+  }
+
+  Widget _crossFade(Widget first, Widget second, bool isExpanded) {
+    return new Stack(
+      alignment: FractionalOffset.center,
+      children: <Widget>[
+        new AnimatedOpacity(
+          opacity: isExpanded ? 0.0 : 1.0,
+          duration: const Duration(milliseconds: 200),
+          curve: _flipCurves(isExpanded),
+          child: first
+        ),
+        new AnimatedOpacity(
+          opacity: isExpanded ? 1.0 : 0.0,
+          duration: const Duration(milliseconds: 200),
+          curve: _flipCurves(!isExpanded),
+          child: second
+        )
+      ]
+    );
+  }
+
+  @override
+  Widget build(BuildContext context) {
+    return new Row(
+      children: <Widget>[
+        new Flexible(
+          flex: 2,
+          child: new Container(
+            margin: const EdgeInsets.only(left: 24.0),
+            child: new DefaultTextStyle(
+              style: const TextStyle(
+                color: Colors.black87,
+                fontSize: 15.0
+              ),
+              child: new Text(name)
```

Comment:
You don't need a inherited widget (DefaultTextStyle) here, you can just specify the Text style: parameter.